OWNERS.md

```diff
@@ -20,4 +18,5 @@ In alphabetical order:
 - Marco Palladino ([subnetmarco](https://github.com/subnetmarco)) (marco@konghq.com)
 - Nikolay Nikolaev ([nickolaev](https://github.com/nickolaev)) (nikolay.nikolaev@konghq.com)
 - Paul Parkanzky ([parkanzky](https://github.com/parkanzky)) (paul.parkanzky@konghq.com)
+- Philipp Rudloff ([kleinfreund](https://github.com/kleinfreund)) (philipp.rudloff@konghq.com)
 - Tomasz Wylężek ([tomaszwylezek](https://github.com/tomaszwylezek)) (tomwylezek@gmail.com)
```

Comment:
This page should really be extracted for github.com/kumahq/.github

Reply:
Isn’t this list project-specific?

Reply:
I don't know I'd rather have an OWNERS once but up to date rather than OWNERS everywhere but really out of date :)

Reply:
Filed an issue for this to tackle this separately: https://github.com/kumahq/kuma-gui/issues/426